bnd-maven-plugin/.gitignore

```diff
@@ -1 +1,4 @@
+/.classpath
+/.project
+/.settings/
```

Comment:
How is this related to the m2e change? perhaps we should put eclipse metadata under source control like other bnd projects.